### PR TITLE
java_binary should line up with kotlin run markers.

### PR DESCRIPTION
**Pull request by hsyed**
Simple fix for #387 -- this regressed or it was never included from my kotlin-v3 work.

### CODE REVIEW

**Comment by hsyed** [comment]
@chaoren @brendandouglas could you have a look and upstream this simple upstream this simple PR please.

File: kotlin/src/com/google/idea/blaze/kotlin/run/producers/BlazeKotlinRunConfigurationProducer.java [hunk 1]
```diff
@@ -152,6 +152,6 @@ private static FilteredTargetMap computeTargetMap(Project project, BlazeProjectD
         project,
         projectData.artifactLocationDecoder,
         projectData.targetMap,
-        (target) -> target.kind == Kind.KT_JVM_BINARY && target.isPlainTarget());
+        (target) -> (target.kind == Kind.KT_JVM_BINARY || target.kind == Kind.JAVA_BINARY) && target.isPlainTarget());
```
Comment (chaoren) [inline kotlin/src/com/google/idea/blaze/kotlin/run/producers/BlazeKotlinRunConfigurationProducer.java]:
```
KotlinRunConfigurationProducer.Companion.getEntryPointContainer(location.getPsiElement());
```
above should return `null` for `java_binary`s without a kotlin main method, correct?

Just making sure this wouldn't show up for regular `java_binary`s.

**Comment by chaoren** [comment]
Merged upstream.

File: kotlin/src/com/google/idea/blaze/kotlin/run/producers/BlazeKotlinRunConfigurationProducer.java (hunk 1 above)
Comment (hsyed) [reply kotlin/src/com/google/idea/blaze/kotlin/run/producers/BlazeKotlinRunConfigurationProducer.java]: correct, and thanks !

**Pull request closed without merging**